### DISCUSSION TITLE
flashing via ESP Flasher - Windows

### DISCUSSION THREAD

**Discussion by Panoru84** [frickelzeugs/FingerprintDoorbell#38] [Ideas]
Hi all,

as I'm not running linux but have the windows GUI version of ESP Flasher already in use I gave it a try.

It took me some time but finally with the following settings I've been able to flash my ESP32 with the Windows tool.

Maybe somebody who knows more about this can check this for the latest version of ESP Flasher and this can be added to the documentation because it makes flashing really very easy.  I wasn't able to flash it with the latest version but v3.8.8 works fine.

I took the memory adresses from the linux command and translated the spiffs.bin one.

![ESP-Flasher](https://user-images.githubusercontent.com/37895666/201983354-3cae39fd-a5b3-4d7c-925b-482be7ca0d6b.png)

**Comment by frickelzeugs** [comment]
Hi thanks for your Feedback but I'm on windows too and the instructions on the site work here as well. All you need is a a running python installation (first step of the instructions), then all commands work as described in the instructions on the command line.
